src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -418,34 +417,15 @@ impl<S: StateStore, SD: ValueRowSerde> StorageTableInner<S, SD> {
             _ => CachePolicy::Fill(CachePriority::High),
         };
 
-        let raw_key_ranges = if !ordered
-            && matches!(encoded_key_range.start_bound(), Unbounded)
-            && matches!(encoded_key_range.end_bound(), Unbounded)
-        {
-            // If the range is unbounded and order is not required, we can create a single iterator
-            // for each continuous vnode range.
```

Comment:
I guess removing this can significantly hurt the performance for a full scan on a small table (like Top-10 or simple aggregation).

Reply:
🤔 I will add a session varible to control it but I am not sure about which default value should be

Reply:
or can we get some table's cardinarity statics here or in optimizer? c.c. @hzxa21 @wcy-fdu

Reply:
We have maintained the physical key count in meta. Currently CN doesn't have a way to get the key count stats but we can add it if needed.